registry_no_code_module.go

```diff
@@ -0,0 +1,266 @@
+package tfe
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+)
+
+// Compile-time proof of interface implementation.
+var _ RegistryNoCodeModules = (*registryNoCodeModules)(nil)
+
+// RegistryNoCodeModules describes all the registry no-code module related methods that the Terraform
+// Enterprise API supports.
+//
+// TFE API docs: (TODO: Add link to API docs)
+type RegistryNoCodeModules interface {
+
+	// Create a registry no-code module
+	// **Note: This API is still in BETA and subject to change.**
+	Create(ctx context.Context, organization string, options RegistryNoCodeModuleCreateOptions) (*RegistryNoCodeModule, error)
+
+	// Read a registry no-code  module
+	// **Note: This API is still in BETA and subject to change.**
+	Read(ctx context.Context, noCodeModuleID string, options *RegistryNoCodeModuleReadOptions) (*RegistryNoCodeModule, error)
+
+	// Update a registry no-code module
+	// **Note: This API is still in BETA and subject to change.**
+	Update(ctx context.Context, noCodeModuleID string, options RegistryNoCodeModuleUpdateOptions) (*RegistryNoCodeModule, error)
+
+	// Delete a registry no-code module
+	// **Note: This API is still in BETA and subject to change.**
+	Delete(ctx context.Context, ID string) error
+}
+
+// registryNoCodeModules implements RegistryNoCodeModules.
+type registryNoCodeModules struct {
+	client *Client
+}
+
+// RegistryNoCodeModule represents a registry no-code module
+type RegistryNoCodeModule struct {
+	ID         string `jsonapi:"primary,no-code-modules"`
+	VersionPin string `jsonapi:"attr,version-pin,omitempty"`
+	Enabled    bool   `jsonapi:"attr,enabled,omitempty"`
```

Comment:
I don't think these need omitempty but also it would probably not cause a problem.

```suggestion
	VersionPin string `jsonapi:"attr,version-pin"`
	Enabled    bool   `jsonapi:"attr,enabled"`
```